cpu/numactl.py

```diff
@@ -67,6 +74,28 @@ def setUp(self):
 
         build.make(self.sourcedir)
 
+        interfaces = netifaces.interfaces()
+        interface = self.params.get("interface")
+        if interface not in interfaces:
+            self.cancel("%s interface is not available" % interface)
+        self.iface = interface
```

Comment:
@FarooqAbdulla01 while reading the `interface` make it as global like 
`self.iface = self.params.get("interface", default=None)`
just to avoid extra variable
Add default value as None